clients/src/main/java/org/apache/kafka/common/utils/SystemTime.java

```diff
@@ -26,6 +26,18 @@
  */
 public class SystemTime implements Time {
 
+    /**
+     * use the static inner class to implement the singleton pattern
+     * to avoid the overhead of synchronization
```

Comment:
Can you explain this comment? I don't know what it's referring to, as none of this code is synchronized.

The static inner class seems to be a lazy initialization trick, which is ineffective here (due to Time.SYSTEM) and unneeded (since the SystemTime constructor is very cheap).

Reply:
True, `SystemTime` is very cheap, but I think that don't use double-checked locking to implement the singleton pattern, maybe I should remove this comment is more better.